src/main/md/otio-core.md

```diff
@@ -1775,11 +1775,21 @@ Double (binary64) floating point number as defined in IEEE 754.
   }
 ```
 
-### JSONObject
+### MetadataObject
 
 #### General
 
-A JSON Object as defined at www.json.org.
+A `MetadataObject` instance represents a single `object` as defined at www.json.org.
+
+A `MetadataObject` instance and any of its descendent `object` values may be an instance of
+[`SerializableObject`](#object-model-SerializableObject).
+
+_EXAMPLE_: A `MetadataObject` instance can contain `RationalTime` and `TimeRange` instances.
+
+#### Processing
+
+Unless it is a descendent of an instance of [`SerializableObject`](#object-model-SerializableObject), a processor shall interpret a
+`number` value as a [`Float`](#object-model-Float).
```

Comment:
I'm not sure I fully understand this. Is this saying all numbers should be interpreted as float values unless they are part of a `SerializableObject`?

Reply:
`Float` type in the specification is defined as a IEEE double. I can see how this might be confusing. What about changing the name of the `Float` type to `Double`?

Reply:
@reinecke The idea is that the interpretation of _JSON numbers_ as used by `SerializableObject` and its descendants, is determined by the object model, e.g. they are sometimes interpreted as integers. For other _JSON numbers_ not used by `SerializableObject` and its descendants, they are interpreted as _IEEE double_. Makes sense?

Reply:
Okay, that makes sense. I think I misinterpreted the context as being part of metadata object when it is a separate section alltogether.

On the point of float vs. double: I agree about switching to Double. This matches the historical context best.

Reply:
@reinecke Changed floating point type name to `Double`

Reply:
I was just reviewing one more time and the sentence still feels hard to parse the meaning of. Perhaps we could try wording like:
```
A processor shall interpret a `number` value as a [`Double`](#object-model-Float) unless it is a descendent of an instance of [`SerializableObject`](#object-model-SerializableObject) and the schema dictates a different interpretation.
```

One other detail is that in the OTIO implementations, numbers without the fraction (`.`) or exponent (`e`|`E`) delimiters are interpreted as `int`. Is this something we should call out here?

Reply:
> One other detail is that in the OTIO implementations, numbers without the fraction (.) or exponent (e|E) delimiters are interpreted as int. Is this something we should call out here?

Is this a requirement of all implementations? If so, it needs to be called out.

Reply:
I can't think of a reason it has to work this way, it's really up to the consumer what representation is in memory I suppose.